### PR TITLE
Card.Media: add support for videos objects

### PR DESCRIPTION
**Pull request by xjuan**
https://phabricator.endlessm.com/T16868

### CODE REVIEW

File: js/app/modules/contentGroup/mediaLightbox.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ const MediaLightbox = new Module.Class({
         if (this.lightbox_widget instanceof EosKnowledgePrivate.MediaBin) {
             this.lightbox_widget.stop();
         }
+        this.drop_submodule(this.lightbox_widget);
```
Comment (tchx84) [inline js/app/modules/contentGroup/mediaLightbox.js]: looks like this change belongs to the previous commit ;)

**Comment by tchx84** [comment]
@xjuan one more thing, 6 tests are failing.

File: js/app/modules/contentGroup/mediaLightbox.js (hunk 1 above)
Comment (xjuan) [inline js/app/modules/contentGroup/mediaLightbox.js]: Not really, in the previous commit I added the dropped() method not drop_submodule()

**Comment by xjuan** [comment]
Fixed the NULL bug and posted a test shard in phab

**Comment by xjuan** [comment]
You will have to  remove
```        if (this._current_index === -1)
            return;```
from ContentGroup.MediaLightbox _preview_media_object() function for the test shard to work
since the video is not listed as a resource of the article

**Comment by tchx84** [comment]
@xjuan 5 tests are still failing

**Comment by xjuan** [comment]
Tests fixed again!

**Pull request merged**